src/library/export/coverartcopyworker.cpp

```diff
@@ -0,0 +1,102 @@
+#include "library/export/coverartcopyworker.h"
+
+#include <QBuffer>
+#include <QDebug>
+#include <QFileInfo>
+#include <QMessageBox>
+
+void CoverArtCopyWorker::run() {
+    copyFile(m_coverArtImage, m_coverArtCopyPath);
+
+    if (m_bStop.loadAcquire()) {
```

Comment:
When is this code used? Is there a way to cancel this single file operation?

Reply:
I am sorry I couldn't fully understand, what exactly means `cancel single file operation`?

Reply:
Does the code implement a feature, or is it a left-over? That was my original question.

Reply:
I don't think that is a left over IMHO. That starts the copying cover art. It overrides the QThread's run() method. Could you please correct me if I am wrong.